airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/argproviders/DataArgumentsProvider.kt

```diff
@@ -43,6 +43,7 @@ class DataArgumentsProvider : ArgumentsProvider {
     }
 
     companion object {
+        @JvmField
```

Comment:
I thought we wanted to avoid jvmfield? (from https://www.notion.so/Converting-the-Java-CDK-to-Kotlin-a29997b7bc7b40c689588e6ce4b1bb66)

Reply:
To facilitate same pattern like `ClassName.CONSTANT_STATIC_FIELD`. without this it'll look like `ClassName.Companion.getCONSTANT_STATIC_FIELD()`